NAMESPACE

```diff
@@ -22,33 +26,58 @@ export(dataset_shuffle)
 export(dataset_shuffle_and_repeat)
 export(dataset_skip)
 export(dataset_take)
+export(dataset_use_spec)
 export(dataset_window)
 export(delim_record_spec)
+export(dense_features)
+export(ends_with)
+export(everything)
+export(feature_spec)
 export(file_list_dataset)
+export(fit)
 export(fixed_length_record_dataset)
+export(has_type)
 export(input_fn.tf_dataset)
 export(install_tensorflow)
 export(iterator_get_next)
 export(iterator_initializer)
 export(iterator_make_initializer)
 export(iterator_string_handle)
+export(layer_input_from_dataset)
 export(make_csv_dataset)
 export(make_iterator_from_string_handle)
 export(make_iterator_from_structure)
 export(make_iterator_initializable)
 export(make_iterator_one_shot)
+export(matches)
 export(next_batch)
+export(num_range)
+export(one_of)
 export(out_of_range_handler)
 export(output_shapes)
 export(output_types)
 export(range_dataset)
 export(read_files)
 export(sample_from_datasets)
+export(scaler_min_max)
+export(scaler_standard)
 export(shape)
 export(sparse_tensor_slices_dataset)
 export(sql_dataset)
 export(sql_record_spec)
 export(sqlite_dataset)
+export(starts_with)
+export(step_bucketized_column)
+export(step_categorical_column_with_hash_bucket)
+export(step_categorical_column_with_identity)
+export(step_categorical_column_with_vocabulary_file)
+export(step_categorical_column_with_vocabulary_list)
+export(step_crossed_column)
+export(step_embedding_column)
+export(step_indicator_column)
+export(step_numeric_column)
+export(step_remove_column)
+export(step_shared_embeddings_column)
 export(tensor_slices_dataset)
 export(tensors_dataset)
 export(text_line_dataset)
```

Comment:
Can we rename all the tensor conversion functions to have the same prefix? For instance:

```
as_slices_dataset()
as_tensor_dataset()
as_text_dataset()
```

Do we really need to export `as_tf_dataset()`? Would users use this directly? Feels to me like this should be `@keywords internal` since I wouldn't expect users to have to add classes

Reply:
@dfalbel mentioned these functions have been in `tfdatasets` for a while... so worth considering the rename (while not breaking compatibility and issue a deprecated warning), but not in this PR.